azure.yaml

```diff
@@ -25,35 +27,35 @@
       componentVersion: 0.18.1
       version: 1.2.0
   authorities:
-  - name: app-operator
-    version: 1.0.0
-  - name: azure-operator
-    version: 2.9.0
-  - name: cert-operator
-    version: 0.1.0
-  - name: chart-operator
-    version: 0.7.0
-  - name: cluster-operator
-    provider: azure
-    version: 0.23.0
-  date: 2020-01-29T12:00:00Z
-  version: 11.1.0
-- active: true
-  authorities:
-  - name: app-operator
-    version: 1.0.0
-  - name: azure-operator
-    version: 2.8.0
-  - name: cert-operator
-    version: 0.1.0
-  - name: chart-operator
-    version: 0.7.0
-  - name: cluster-operator
-    provider: azure
-    version: 0.21.4
+    - name: app-operator
```

Comment:
I found out these parts need an indentation. Not sure how it was working before.